### DISCUSSION TITLE
Stability issues after 1.1.0

### DISCUSSION THREAD

**Discussion by simon1176f** [apohl79/audiogridder#520] [Help]
1.1.0 is working incredibly well as LPX as the host and 4 x Win10 servers. Occasionally have to restart servers. 1.1.1 and 1.2.0 are unusable here with CPU spikes and kernal crashes every 5 or so minutes (Catalina (host) completely freezing and restarting, narrowed it down to AG plugin). Again, 1.1.0 is perfect. Cool new ideas but are old sessions backwards compatable with 1.1.0?

**Comment by apohl79** [comment]
As 1.1.1 introduced no code changes from 1.1.0, your problems are likely related to some other factor, like the JUCE version. So the only way to figure out whats going on, is if you submit full bug reports (https://audiogridder.com/bug-reports/) including core dumps if possible.

**Comment by simon1176f** [comment]
Does AG install all the current versions of files it needs? JUCE etc.
Thankyou for your endless work, you've made things great for us. The ver
1.2 looks amazing. My (host mac) computer does full freeze and reset under
1.1.1 or  1.2.0. approx 5 mins after project start in LPX. 1.1.0 is
perfect. I will submit a report

**Reply by apohl79** [reply]
> Does AG install all the current versions of files it needs? JUCE etc.

Yes you don't have to worry about this. It's all included in each release.

> Thankyou for your endless work, you've made things great for us.

Thx :-)
